blocks/cards/cards.css

```diff
@@ -24,8 +48,67 @@
     margin-top: 0;
 }
 
-.cards > ul > li img {
-    width: 100%;
-    aspect-ratio: 4 / 3;
-    object-fit: cover;
+.cards .cards-card-body h5 {
+    font-family: 'Roboto', sans-serif;
+    font-size: var(--heading-font-size-s);
+    line-height: 1.2;
+    margin-bottom: 1rem;
+    font-weight: 500;
+    letter-spacing: .4px;
+    line-height: 1.2;
+}
+
+.cards .cards-card-body p {
+    font-size: 1.3rem;
+    margin: 0;
+    font-family: 'Roboto Condensed', sans-serif;
```

Comment:
we should probably make this a variable in styles.css

Reply:
Done